src/artemis/parameters.py

```diff
@@ -48,7 +48,7 @@ class FullParameters:
         undulator_gap=1.0,
         pixels_per_micron_x=None,
         pixels_per_micron_y=None,
-        upper_left=Point2D(x=None, y=None),
+        upper_left=Point3D(x=None, y=None, z=None),
```

Comment:
**should** can you put a comment here about how we're sort of cheating here and it should actually be 2x 2D upper lefts. Also can you comment that it is in pixels